### PR TITLE
OCLOMRS-227 : All the pages in the App to have consistence margin and padding

### PR DESCRIPTION
**Pull request by alexmochu**
# JIRA TICKET NAME:
[All the pages in the App to have consistence margin and padding](https://issues.openmrs.org/browse/OCLOMRS-227)

# Summary:
The margin and padding on the pages should consistency. All the pages should have a uniform margin and padding not to have other pages with a bigger, other smaller and other none.

### CODE REVIEW

File: src/components/userDasboard/components/DashboardDetails.jsx [hunk 1]
```diff
@@ -14,9 +14,6 @@ const DashboardDetails = ({ numberOfOrgs, numberOfDictionary, organizations }) =
             the traditional OCL
           </a>.
         </p>
-        <p className="lead">
-          You can <Link to="dashboard/dictionaries">view all OpenMRS public dictionaries</Link>.
```
Comment (dondrzzy) [inline src/components/userDasboard/components/DashboardDetails.jsx]: Are these deletions part of the ticket?

File: src/components/dashboard/components/dictionary/styles/dictionary-modal.css [hunk 2]
```diff
@@ -17,6 +17,8 @@
     padding-right: 8%;
     margin-left: 4%;
 }
+
```
Comment (kmunene) [inline src/components/dashboard/components/dictionary/styles/dictionary-modal.css]: What happened here?

**Comment by alexmochu** [comment]
@dkayiwa The same applies here. It requires this PR [Use SASS for Styling](https://github.com/openmrs/openmrs-ocl-client/pull/159) by @muhozi to be merged first so that I can pull those SASS changes.

**Comment by dkayiwa** [comment]
@alexmochu looks like you should have worked on the other ticket, instead of doing this and wait for a week.

File: src/components/userDasboard/components/DashboardDetails.jsx (hunk 1 above)
Comment (alexmochu) [reply src/components/userDasboard/components/DashboardDetails.jsx]: This was not part of the task scope, I have removed it.

File: src/components/dashboard/components/dictionary/styles/dictionary-modal.css (hunk 2 above)
Comment (alexmochu) [reply src/components/dashboard/components/dictionary/styles/dictionary-modal.css]: I have removed that extra white space and updated the PR.

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export class AddBulkConcepts extends Component {
     const dictionaryName = localStorage.getItem('dictionaryName');
     const { cielConcepts } = this.props;
     return (
-      <div className="container add-bulk-concepts">
```
Comment (kmunene) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: why did you remove the container class?

File: src/components/bulkConcepts/container/BulkConceptsPage.jsx [hunk 4]
```diff
@@ -110,7 +110,7 @@ export class BulkConceptsPage extends Component {
     const lastConcept = indexOfFirstConcept + currentConcepts.length;
     const firstConcept = indexOfLastConcept - 19;
     return (
-      <div className="container custom-dictionary-concepts bulk-concepts">
+      <div className="bulk-concepts">
```
Comment (kmunene) [inline src/components/bulkConcepts/container/BulkConceptsPage.jsx]: Same here.

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 3 above)
Comment (alexmochu) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: I have removed the container class since it was aligning that 'div' to the center by default which should not be the case here.

File: src/components/bulkConcepts/container/BulkConceptsPage.jsx (hunk 4 above)
Comment (alexmochu) [reply src/components/bulkConcepts/container/BulkConceptsPage.jsx]: The same apples here. I have removed the container class since it was aligning that 'div' to the center by default which should.

File: src/styles/dictionary_concepts.scss [hunk 5]
```diff
@@ -1,6 +1,20 @@
...
   margin-top: -3%;
+  padding-right: 8%;
+  margin-left: 4%;
+}
+
+.bulk-concepts {
+  text-align: left;
+  margin-top: -3%;
+  padding-right: 4%;
+  margin-left: 5%;
```
Comment (Duncanian) [inline src/styles/dictionary_concepts.scss]: Is the margin applicable to all media?

File: src/styles/dictionary_concepts.scss [hunk 6]
```diff
@@ -1,6 +1,20 @@
...
+}
+
+.all-dictionaries {
+  margin-right: -8%;
+  margin-left: -8%;
```
[the remark this answers is not in the document]
Comment (alexmochu) [reply src/styles/dictionary_concepts.scss]: @dondrzzy That is a very good suggestion, but this works fine. I would recommend having another ticket to handle the CSS for all media.

File: src/styles/dictionary_concepts.scss (hunk 5 above)
Comment (alexmochu) [reply src/styles/dictionary_concepts.scss]: @Duncanian In this case I was just handling inconsistency of margin and padding in desktop layout.

File: src/styles/bulk_concepts.scss [hunk 7]
```diff
@@ -24,6 +24,8 @@ legend.scheduler-border {
   text-align: left !important;
   float: center;
   margin-top: -3%;
+  padding-right: 5%;
```
[the remark this answers is not in the document]
Comment (alexmochu) [reply src/styles/bulk_concepts.scss]: That was not in the scope of the task

File: src/components/bulkConcepts/container/BulkConceptsPage.jsx (hunk 4 above)
Comment (alexmochu) [reply src/components/bulkConcepts/container/BulkConceptsPage.jsx]: Same applies here. I have replaced container with container-fluid.

**Comment by dkayiwa** [comment]
@alexmochu why is this still open?

**Comment by alexmochu** [comment]
@dkayiwa I was waiting to be approved by all the team members so that I can request for your review and approval. The PR has now been approved by the whole team and is ready to be merged.

**Comment by dkayiwa** [comment]
@alexmochu did you report to me when the team did not review it after a full day?

**Comment by alexmochu** [comment]
@dkayiwa While waiting for the [OCLOMRS-240](https://github.com/openmrs/openmrs-ocl-client/pull/159) to be merged, I had already started working on [OCLOMRS-229](https://github.com/openmrs/openmrs-ocl-client/pull/165) and [OCLOMRS-231](https://github.com/openmrs/openmrs-ocl-client/pull/157)

**Comment by dkayiwa** [comment]
@alexmochu that does not look like an answer to my question.

**Comment by alexmochu** [comment]
@dkayiwa That was a response to a question you had left earlier. 
In regards to reaching out to you, I felt it was necessary for me to first exhaust reaching out to my teammates before going to you. Also since the team members had already reviewed and left feedback the only thing left was to approve after I had implemented the feedback.

File: src/styles/dictionary_concepts.scss (hunk 6 above)
[the remark this answers is not in the document]
Comment (dkayiwa) [reply src/styles/dictionary_concepts.scss]: @alexmochu did you create the ticket?
